### PR TITLE
Set ghcr.io username to user triggering the build

### PR DESCRIPTION
**Pull request by HubbeKing**
This should hopefully make pushes work in PRs? I'm grasping at straws at this point.

### CODE REVIEW

**Review by ekimekim — APPROVED** [review]
Let's give it a try

**Comment by HubbeKing** [comment]
So this appears to work. I'm unsure how the permissions work themselves out, but I'm *reasonably* sure that once this is merged, anyone with write access to the repo can open a PR and the workflow will build&push the images. No idea what happens if a Random Nobody opens a PR.

The documentation doesn't really say anything...

**Pull request merged**
